### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a few typos I spotted in the documentation of the project.

### CODE REVIEW

File: annotations/src/main/java/com/google/errorprone/annotations/RestrictedApi.java [hunk 1]
```diff
@@ -103,7 +103,7 @@
    *
    * <p>This should only be used if callers should aggressively move away from this API (or change
    * to a allowlist annotation after review). Too many warnings will lead to ALL warnings being
-   * ignored, so tread very carefully.
+   * ignored, so treat very carefully.
```
Comment (Stephan202) [inline annotations/src/main/java/com/google/errorprone/annotations/RestrictedApi.java:106]: I think that the original was [meant to be that way](https://plainenglish.com/expressions/tread-carefully/) :)

File: docs/bugpattern/ProtocolBufferOrdinal.md [hunk 2]
```diff
@@ -13,5 +13,5 @@ You should very likely use `getNumber()` in preference to `ordinal()` in all
 circumstances since it's a more stable value.
 
 Note: If you're changing code that was already using ordinal(), it's likely that
-getNumber() will return a different real value. Tread carefully to avoid
+getNumber() will return a different real value. Treat carefully to avoid
```
Comment (Stephan202) [inline docs/bugpattern/ProtocolBufferOrdinal.md:16]: Likewise here :)

File: annotations/src/main/java/com/google/errorprone/annotations/RestrictedApi.java (hunk 1 above)
Comment (deining) [reply annotations/src/main/java/com/google/errorprone/annotations/RestrictedApi.java]: Reverted to the original version, sorry for the noise!

File: docs/bugpattern/ProtocolBufferOrdinal.md (hunk 2 above)
Comment (deining) [reply docs/bugpattern/ProtocolBufferOrdinal.md]: Reverted to the original version, sorry for the noise!

**Review by Stephan202 — APPROVED** [review]
I'm not a maintainer, but LGTM!

**Review by cushon — APPROVED** [review]

**Comment by cushon** [comment]
Thanks for the fixes!

**Review by cushon — APPROVED** [review]

**Pull request closed without merging**